rhel/matcher.go

```diff
@@ -25,15 +28,37 @@ func (*Matcher) Filter(record *claircore.IndexRecord) bool {
 }
 
 // Query implements driver.Matcher.
-func (*Matcher) Query() []driver.MatchConstraint {
-	return []driver.MatchConstraint{
-		driver.PackageModule,
-		driver.RepositoryName,
+func (m *Matcher) Query() []driver.MatchConstraint {
+	mcs := []driver.MatchConstraint{driver.PackageModule}
```

Comment:
We are not deriving any distribution info from the VEX data. The OVAL updater didn't derive distribution from the data either, it parses the name of the file, and this info isn't used for matching.

Reply:
This is used for the `WHERE` clause, right? I think I'm still confused why we don't also filter out anything under a different OS

Reply:
It would be used in the where clause if we added a constraint here, but we currently don't have a good way to derive the distro from the VEX data (AFAICT) so we can't filter on it. Currently with OVAL data we derive the distro from the OVAL file's name but it isn't used in the matching.